src/transform/ThemeTransform.css

```diff
@@ -266,6 +270,87 @@ styles with only the things which need tweaking.
 .pagelib_theme_dark .content .mw-highlight span.vm { color: #40ffff } /* Name.Variable.Magic */
 .pagelib_theme_dark .content .mw-highlight span.il { color: #3677a9 } /* Literal.Number.Integer.Long */
 
+/* Same 'mw-highlight' colors for dark background are used for black too */
+/* From: https://raw.githubusercontent.com/richleland/pygments-css/master/native.css */
+.pagelib_theme_black .content .mw-highlight span.hll { background-color: #404040 }
+/* .pagelib_theme_black .content .mw-highlight  { background: #202020; color: #d0d0d0 } */
```

Comment:
This does not seem to be commented out in the original or dark mode. Is it intentional? If so, why?

Reply:
Oh this was intentional. The background (for black) is a bit further up:
https://github.com/wikimedia/wikimedia-page-library/pull/106/files#diff-340d93103cd4c516e89429d258b93b35R189

But I left these in place and commented out in case we ever re-sync with the upstream `pygments` classes as reminders that this line isn't wanted. We're using all of the other lines otherwise.

Reply:
I'm open to not doing that ^ and just removing these lines and instead adding a comment instead.

Reply:
Ah, makes sense! A comment would be welcome at an opportune time but it's nothing to fret about.